src/components/FetchArticles.tsx

```diff
@@ -24,10 +24,13 @@ const FetchArticles: FunctionComponent<Props> = ({ articleType }) => {
 
   useEffect(() => {
     const loadArticles = async () => {
-      const data = await getArticles(startEntries, articleType).catch((err) => {
-        console.log(err)
-        setError(err)
-      })
```

Comment:
@JasonFritsche there was a catch on the await (which is strange - hence why the nullable) so I just removed that and added a check if the return from the `articlesApi` is an error or not.